playbooks/roles/foreman_repositories/tasks/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
-- include: release_repos.yml
-  when: foreman_repositories_use_release and not foreman_repositories_use_koji
+- include: debian_release_repos.yml
```

Comment:
How about `include: "{{ ansible_os_family }}_release_repos.yml"`? allows us to fold both families into one include line with fewer `when` statements.